diffuzzer/src/abstract_fs/trace.rs

```diff
@@ -93,14 +99,19 @@ impl Trace {
         }
         Ok(trace)
     }
-    pub fn same_as(&self, other: &Trace) -> bool {
-        self == other
-    }
+
     pub fn has_errors(&self) -> bool {
         self.rows.iter().any(|row| row.errno.code != 0)
     }
 }
 
+impl TraceRow {
+    pub fn to_index_independent_form(mut self) -> TraceRow {
```

Comment:
```suggestion
    pub fn ignore_index(mut self) -> TraceRow {
```
Возможно лучше возвращать новый TraceRow

Reply:
Думаю просто сделаю метод сравнения без индексов и удалю это